platforms/hyperledger-indy/configuration/samples/network-indy-newnode.yaml

```diff
@@ -0,0 +1,132 @@
+      # the cluster has already been created.
+      k8s:
+        config_file: "cluster_config"
+        context: "kubernetes-admin@kubernetes"
+
+      # Hashicorp Vault server address and root-token. Vault should be unsealed.
+      # Do not check-in root_token
+      vault:
+        url: "vault_addr"
+        root_token: "vault_root_token"
+
+      # Git Repo details which will be used by GitOps/Flux.
+      # Do not check-in git_access_token
+      gitops:
+        git_ssh: "ssh://git@github.com/<username>/blockchain-automation-framework.git"                   # Gitops ssh url for flux value files 
+        branch: "develop"                     # Git branch where release is being made
+        release_dir: "platforms/hyperledger-indy/releases/dev"           # Relative Path in the Git repo for flux sync per environment. 
+        chart_source: "platforms/hyperledger-indy/charts"               # Relative Path where the Helm charts are stored in Git repo
+        git_push_url: "github.com/<username>/blockchain-automation-framework.git"             # Gitops https URL for git push 
+        username: "git_username"                    # Git Service user who has rights to check-in in all branches
+        password: "git_access_token"                    # Git Server user password
+        email: "git_email"                          # Email to use in git config
+        private_key: "path_to_private_key"          # Path to private key file which has write-access to the git repo
+
+      # Services maps to the pods that will be deployed on the k8s cluster
+      # This sample has trustee, 2 stewards and endoorser
+      services:
+        trustees:
+        - trustee:
+          name: provider-trustee
+          genesis: true
+        stewards:
+        - steward:
+          name: provider-steward-1
+          type: VALIDATOR
+          genesis: true
+          publicIp: 3.221.78.194    # IP address of current organization in current availability zone
+          node:
+            port: 9711
+            targetPort: 9711
+            ambassador: 9711        # Port for ambassador service
+          client:
+            port: 9712
+            targetPort: 9712
+            ambassador: 9712        # Port for ambassador service
+        - steward:
+          name: provider-steward-2
+          type: VALIDATOR
+          genesis: true
+          publicIp: 3.221.78.194    # IP address of current organization in current availability zone
+          node:
+            port: 9721
+            targetPort: 9721
+            ambassador: 9721        # Port for ambassador service
+          client:
+            port: 9722
+            targetPort: 9722
+            ambassador: 9722        # Port for ambassador service
+        endorsers:
+        - endorser:
+          name: provider-endorser
+          full_name: Some Decentralized Identity Mobile Services Provider
+          avatar: http://provider.com/avatar.png
+  
+      
```

Comment:
new line should be there and extra tabbled/spaced lines should be removed from the end